apps/oxfmt/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "oxfmt",
+  "version": "0.5.0",
+  "type": "module",
+  "main": "dist/index.js",
+  "bin": "dist/cli.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "build": "pnpm run build-napi-release && pnpm run build-js",
+    "build-dev": "pnpm run build-napi && pnpm run build-js",
+    "build-napi": "napi build --esm --platform --js ./bindings.js --dts ./bindings.d.ts --output-dir src-js --no-dts-cache",
+    "build-napi-release": "pnpm run build-napi --release --features allocator",
+    "build-js": "node scripts/build.js",
+    "test": "tsc && vitest --dir ./test run"
+  },
+  "engines": {
+    "node": "^20.19.0 || >=22.12.0"
+  },
+  "description": "Formatter for the JavaScript Oxidation Compiler",
+  "author": "Boshen and oxc contributors",
+  "license": "MIT",
+  "homepage": "https://oxc.rs",
+  "bugs": "https://github.com/oxc-project/oxc/issues",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/oxc-project/oxc.git",
+    "directory": "apps/oxfmt"
+  },
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org/",
+    "access": "public"
+  },
+  "files": [
+    "dist"
+  ],
+  "devDependencies": {
+    "execa": "^9.6.0",
+    "prettier": "^3.4.2",
```

Comment:
- This should be `^3.6.2`(= latest) and/or pinned?
- This should be `dependencies`?